.github/workflows/vercel-deployment.yml

```diff
@@ -7,18 +7,17 @@ env:
 
 on:
   push:
-    branches-ignore:
-      - main
+
 jobs:
-  Deploy-Preview:
+  Deploy-Production:
```

Comment:
deploy-prd

create a separate reusable github workflow with a parameter environment = dev, uat, prd

dev = some site with the hash
uat = preview
prd = production

Reply:
3 is basically this 
https://github.com/ai-cfia/finesse-frontend/pull/45#discussion_r1406819736
we can forget about the second point